CollapseLauncher/XAMLs/MainApp/Pages/FileCleanupPage.xaml.cs

```diff
@@ -220,9 +344,10 @@ private async Task PerformRemoval(ICollection<LocalFileInfo>? deletionSource, lo
                 await Task.Run(() => RecycleBin.MoveFileToRecycleBin(toBeDeleted));
```

Comment:
Should we do it in parallel as well?

Reply:
For recycle bin there is no need. It passes the list of files to be deleted to Windows default "Move to recycle bin" method then delete it that way